### ISSUE TITLE
[Bug]: New partners should tell you if they won't show up on any sites

### ISSUE THREAD

**Issue by kimadactyl** [geeksforsocialchange/PlaceCal#1272]
## Description

Georgie couldn't work out why a new partner wasn't showing events. It turned out it was because the appropriate tag hadn't been selected. This is a UX bug as it's not at all clear why or what to do about it.

## Steps to reproduce

1. Create a partner outside a current PlaceCal site, or without the right tag, etc
2. Partner will be created and give you a message that it shows up on no sites but not know what to do about it

## What you expected to happen

Basically be a bit more helpful.

- Partners could warn you at creation that they're not going to show up
- The "this doesnt show up on any sites" text could be more helpful

## Screenshots

![WhatsApp Image 2022-06-07 at 10 31 08 AM](https://user-images.githubusercontent.com/511734/172387725-b5f73f93-8269-4c59-969b-07f31c3dfba5.jpeg)

**Comment by ivankocienski** [comment]
Please write the help text you want shown up.
